assets/airports/mkjp.json

```diff
@@ -207,7 +403,34 @@
                 ["DATOM", "ELSER"],
                 ["GUDIL", "MLY"]
             ]
-        }
+        },
+        "KEYNO3": {
+            "icao": "KEYNO3",
+            "name": "Keyno Three",
+            "entryPoints": {
+                "DATOM" : ["DATOM", "R205Q", "_MLY17DA", "_MLY17DB", "_MLY17DC", "_MLY17DD", "_MLY17DE", "_MLY17DF", "_MLY17DG", "_MLY17DH", "_MLY17DI", "_MLY17DI", "_MLY17DJ", "_MLY17DK", "_MLY17DL", "_MLY17DM", "_MLY17DN", "_MLY17DO", "_MLY17DP", "_MLY17DQ", "_MLY17DR", "R290Q", "_MLY17KA", "R300Q", "KEYNO"],
```

Comment:
All of the comments I made regarding the ELSER5 arrival apply to the KEYNO3 as well. I know you mentioned that ELSER5 was the only arrival used, but reading the chart it appears it for the VOR approach into RWY12 while the KEYNO3 is for the ILS approach. I would be surprised if they used the VOR approach over the ILS, but feel free to correct me if I'm wrong!